### PR TITLE
Added solution for Project Euler problem 50.

### PR DESCRIPTION
**Pull request by fpringle**
Name: Consecutive prime sum

The prime 41, can be written as the sum of six consecutive primes:

41 = 2 + 3 + 5 + 7 + 11 + 13
This is the longest sum of consecutive primes that adds to a prime below one-hundred.

The longest sum of consecutive primes below one-thousand that adds to a prime, contains 21 terms, and is equal to 953.

Which prime, below one-million, can be written as the sum of the most consecutive primes?

Reference: [https://projecteuler.net/problem=50](url) 
Reference: #2695

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: project_euler/problem_50/sol1.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Which prime, below one-million, can be written as the sum of the most
+consecutive primes?
+"""
+
+
+def solution(limit: int = 1000000) -> int:
+    """
+    Return the prime below 10^6 that can be written as the sum of the most
+    consecutive primes.
```
Comment (mateuszz0000) [inline project_euler/problem_50/sol1.py:19]: Please add doctests

**Review by mateuszz0000 — CHANGES_REQUESTED** [review]

File: project_euler/problem_50/sol1.py (hunk 1 above)
Comment (fpringle) [reply project_euler/problem_50/sol1.py:19]: Thank you, done!

**Review by dhruvmanila — COMMENTED** [review]
### Announcement:

@fpringle

*This message is sent to all pull requests which are **OPEN, NOT MERGED, not containing 'wontfix' label and contains 'Project Euler' or 'Project' or 'Euler' (case doesn't matter) in their title**. If this message doesn't apply to your pull request, please ignore this.*

### Message:

This is to notify all the users submitting a pull request for the Project Euler solution that the [README.md](https://github.com/TheAlgorithms/Python/blob/master/project_euler/README.md) file has been updated with all the style requirements necessary for the directory and solution files. ***Please read through it and make all the necessary changes required to your submission.***

**Review by 53jk1 — APPROVED** [review]

**Comment by dhruvmanila** [comment]
Thank you for your contribution but we already have another solution for it which was opened and got accepted.

**Pull request closed without merging**